src/node/rpc/member_frontend.h

```diff
@@ -400,6 +401,128 @@ namespace ccf
       return get_path_param(params, "member_id", member_id.value(), error);
     }
 
+    template <typename T>
+    void add_kv_wrapper_endpoint(T table)
+    {
+      constexpr bool is_map = nonstd::is_specialization<T, kv::TypedMap>::value;
+      constexpr bool is_value =
+        nonstd::is_specialization<T, kv::TypedValue>::value;
+
+      if constexpr (!(is_map || is_value))
+      {
+        static_assert(nonstd::dependent_false_v<T>, "Unsupported table type");
+      }
+
+      auto getter =
+        [&, table](endpoints::ReadOnlyEndpointContext& ctx, nlohmann::json&&) {
+          LOG_TRACE_FMT("Called getter for {}", table.get_name());
+          auto response_body = nlohmann::json::object();
+
+          auto handle = ctx.tx.template ro(table);
+          if constexpr (is_map)
+          {
+            handle->foreach([&response_body](const auto& k, const auto& v) {
+              if constexpr (
+                std::is_same_v<typename T::Key, ccf::CodeDigest> ||
+                std::is_same_v<typename T::Key, crypto::Sha256Hash>)
+              {
+                response_body[k.hex_str()] = v;
+              }
+              else
+              {
+                response_body[k] = v;
+              }
+              return true;
+            });
+          }
+          else if constexpr (is_value)
+          {
+            response_body = handle->get();
+          }
+
+          return ccf::make_success(response_body);
+        };
+
+      std::string_view table_name = table.get_name();
+      constexpr auto gov_prefix = "public:ccf.gov.";
+      constexpr auto internal_prefix = "public:ccf.internal.";
+      if (table_name.starts_with(gov_prefix))
+      {
+        table_name.remove_prefix(strlen(gov_prefix));
+      }
+      else if (table_name.starts_with(internal_prefix))
+      {
+        table_name.remove_prefix(strlen(internal_prefix));
+      }
```

Comment:
Note: This exposes the table `public:ccf.gov.foo.bar_baz` at `/gov/kv/foo.bar_baz`. If there's also a `public:ccf.internal.foo.bar_baz`, we'll have a conflict. We think that's fine - we don't have such conflicts and won't add them, if we did we'd prefer to keep the `gov` tables. We want to provide shorter/simpler URIs, so we don't duplicate the `gov` section.

Reply:
Resolved - we no longer expose internal tables like this, so no chance of collision.